fastlane/Fastfile

```diff
@@ -119,3 +119,45 @@ lane :installProfiles do
     FastlaneCore::ProvisioningProfile.install(profile)
   end
 end
+
+desc "Updates project dependencies in Bundler and CocoaPods, then sends a pull request if there are changes. Requires the fastlane-plugin-git_status gem in your Pluginfile."
+lane :updateDependencies do
+    xcversion(version: ENV["XCODE_VERSION"] || '~> 9.4')
```

Comment:
This shouldn't be necessary, as the version is set as a before action.

Reply:
Ah, that action is inside the `ios` lane, so never mind.

Reply:
So this is only to make sure `git` is available, right?

Reply:
Not just `git` but the command-line tools in the case where installing a gem requires building native extensions (looking at you, `nokogiri`).